Add info about omitting the <ending-shape> keyword (#23932)
mdn/content: files/en-us/web/css/gradient/radial-gradient/index.md

    If `<ending-shape>` is specified as `circle`, the size may be given explicitly as a [`<length>`](/en-US/docs/Web/CSS/length), which provides an explicit circle radius. Negative values are invalid.

>     If `<ending-shape>` is specified as `ellipse`, the size may be given as a [`<length-percentage>`](/en-US/docs/Web/CSS/length-percentage) with two values to provide an explicit ellipse size. The first value represents the horizontal radius, the second the vertical radius. Percentages values are relative to the corresponding dimension of the gradient box. Negative values are invalid.

Reviewer: ```suggestion
    If `<ending-shape>` is specified as `ellipse`, the size may be given as a [`<length-percentage>`](/en-US/docs/Web/CSS/length-percentage) with two values to provide an explicit ellipse size. The first value represents the horizontal radius and the second is the vertical radius. Percentage values are relative to the corresponding dimension of the gradient box. Negative values are invalid.
```

>     The `<ending-shape>` keyword may be omitted and the gradient shape will be determined based on the size given. One `<length>` value provides a circle, while two values in `<length-percentage>`units provide an ellipse.

Reviewer: ```suggestion
    When the `<ending-shape>` keyword is omitted, the gradient shape is determined by the size given. One `<length>` value provides a circle, while one `<percentage>` value or two values in `<length-percentage>`units provide an ellipse.
```

I think if the value is a percentage, then it matches the container, therefore an ellipse.

Author: Hi Estelle. Thanks for many valuable suggestions. However, it seems like giving only one `<percentage>` value is not invalid.

[Demo](https://jsfiddle.net/r9kdbLgc/)

Reviewer: Thanks! I think your suggested changes are really helpful. I added a sentence to my original suggestion to let devs know a percent alone is not valid.

Author: Can we mark it as resolved now?
